### DISCUSSION TITLE
how to get item when click dropdown item

### DISCUSSION THREAD

**Discussion by LorenVeri** [AvaloniaUI/Avalonia#8929] [Q&A]
If I binding list object, how to get item in list when click text-block?

```
ListDateTime = new List
{
new DateTimeModel(0, "Hôm qua"),
new DateTimeModel(1, "Hôm nay"),
new DateTimeModel(2, "Tháng trước"),
new DateTimeModel(3, "Đầu tuần tới nay"),
new DateTimeModel(4, "Đầu tháng tới nay"),
new DateTimeModel(7, "7 ngày qua"),
new DateTimeModel(15, "15 ngày qua"),
new DateTimeModel(30, "30 ngày qua"),
new DateTimeModel(90, "90 ngày qua")
};
```
```
<DropDown
  PlaceholderText="Thời gian"
  Name="ListDateTime"
  SelectedItem="{Binding DataContext.SelectDateCommand, ElementName = ListDateTime}"
  Items="{Binding ListDateTime}"
  Height="25"
  VerticalAlignment="Center">
  <DropDown.ItemTemplate>
	  <DataTemplate>
		  <StackPanel>
			  <TextBlock Text="{Binding Name}"/>
		  </StackPanel>
	  </DataTemplate>
  </DropDown.ItemTemplate>
  </DropDown>
```
I want when user click on textblock with element name, i get that item

**Comment by timunie** [comment]
@LorenVeri Are you asking the same question three times? If so, please be noted that this will not speed up things. You are very welcome to Avalonia, and we understand that things may be overhelming at the beginning. Never the less, next time please either ask here in the Discussions section or on Telegram. 

To your question: I think it was already answered in here: https://github.com/AvaloniaUI/Avalonia/issues/8933#issuecomment-1242943490

Maybe you can sum up the solution you finally found for other newbies? That would be great ❤️ 

Thank you and happy coding 
Tim
